force-app/main/default/classes/CountTasksTest.cls

```diff
@@ -0,0 +1,107 @@
+@isTest
+public class CountTasksTest {
+    /*Apex Test Class for all scenarios(insert, update, delete, close task, re open task)*/
+
+    private static final Integer numberOfAccounts = 5;
+    private static final Integer numberOfTasks = 5;
+
+  	@TestSetup
+    private static void makeData(){
+            
+        List<Account> accounts = new List<Account>();
+        
+        Id recordTypeIdUS= Schema.SObjectType.Account.getRecordTypeInfosByDeveloperName().get('United_States').getRecordTypeId();
+        
+        for(Integer i=0; i<numberOfAccounts; i++) {
+            Account acc = new Account(Name = 'Test Account '+i, Type='Prospect', RecordTypeId=recordTypeIdUS);
+            accounts.add(acc);    	
+    	}
+        
+        insert accounts;
+        
+        
+        List<Task> tasks = new List<Task>();
+        for (Account acc : accounts) {
+           for (Integer i = 0; i < numberOfTasks; i++) { 
+            Task tskCall = new Task(Subject='Call', Type='Call', WhatId=acc.Id);
+            tasks.add(tskCall);
+            Task tskEmail= new Task(Subject='Email', Type='Email', WhatId=acc.Id);
+            tasks.add(tskEmail);
+            Task tskMeeting= new Task(Subject='Meeting', Type='Meeting', WhatId=acc.Id);
+            tasks.add(tskMeeting);
+            Task tskOther= new Task(Subject='Other', Type='Other', WhatId=acc.Id);
+            tasks.add(tskOther);
+         
+           }     
+        }
+        insert tasks;
```

Comment:
another, shorter way instead of using tasks.add 
`
insert new List<Task> { tskCall, tskEmail, tskMeeting }; `

Reply:
I've tried to do this (to avoid putting the insert into the for loop):
![image](https://user-images.githubusercontent.com/77618420/216354795-be5ddeb1-d51c-4287-8b85-c443b1551c6b.png)

But it isn't working like it should. Is there something I'm missing?

Reply:
my bad, we are adding several tasks of the same type, in a loop, hence we need a list. For a list with a definite set of named elements, u can use the the suggested way but not here. sorry